### DISCUSSION TITLE
Lens 5 metrics support

### DISCUSSION THREAD

**Discussion by 1001v** [deckhouse/deckhouse#153] [Q&A] [answered]
Hello. I wonder, if there's a way to view cluster metrics in Lens 5? I've tried to fetch it via prometheus operator option by selecting some d8-monitoring services, but no luck. I understand, that in Deckhouse we have full-featured Grafana monitoring, but it's often more comfortable to take a quik look at basic metrics, like Lens provides. Thank you.

**Comment by nabokihms** [comment]
I don't know exactly how Lens works with the Prometheus, but I see three ways to go with:
1. Connect Lens to Deckhouse Prometheus by following this instruction - [connecting-an-external-app-to-prometheus](https://deckhouse.io/en/documentation/v1/modules/300-prometheus/usage.html#connecting-an-external-app-to-prometheus)
2. Expose Prometheus via ingress (like we recommend for additional Grafana instances) - [connecting-prometheus-to-an-external-grafana-instance](https://deckhouse.io/en/documentation/v1/modules/300-prometheus/usage.html#connecting-prometheus-to-an-external-grafana-instance)
3. Use remote write feature to ship metrics from Deckhouse Prometheus to Lens Prometheus with required relabeling rules - [example-of-the-expanded-prometheusremotewrite ](https://deckhouse.io/en/documentation/v1/modules/300-prometheus/usage.html#example-of-the-expanded-prometheusremotewrite)

**Comment by nabokihms** [comment, marked as answer]
We have added instruction about how to connect Deckhouse to Lens -  https://deckhouse.io/en/documentation/v1/modules/300-prometheus/faq.html#how-do-i-get-access-to-prometheus-metrics-from-lens
